assets/src/edit-story/elements/index.js

```diff
@@ -45,10 +46,34 @@ export const createNewElement = (type, attributes = {}) => {
 
 export const createPage = (attributes) => createNewElement('page', attributes);
 
+const defaultBackgroundElement = {
+  type: 'shape',
+  x: PAGE_WIDTH / 4,
+  y: PAGE_HEIGHT / 4,
+  width: PAGE_WIDTH / 3,
+  height: PAGE_HEIGHT / 3,
+  rotationAngle: 0,
+  mask: {
+    type: 'rectangle',
+  },
+  flip: {
+    vertical: false,
+    horizontal: false,
+  },
+  isBackground: true,
+  backgroundColor: {
+    color: { r: 255, g: 255, b: 255, a: 1 },
+  },
+  id: uuidv4(),
+};
+
 export const elementTypes = [
   {
     type: 'page',
-    defaultAttributes: { elements: [] },
+    defaultAttributes: {
+      elements: [defaultBackgroundElement],
```

Comment:
Uh, this is adding by reference - that's a little dangerous. You really should (deep) clone the element (or change it to `getDefaultBackgroundElement()` return a new element every time).  Also, this will make all background elements have the same id.

Reply:
Uf, right 💡 Also, `getDefaultBackgroundElement()` wouldn't help here either as long as it's part of `elementTypes`.

Originally, adding the element was within `addPage` reducer, perhaps it makes more sense there using a `getDefaultBackgroundElement()` util which creates a new element each time.

Anyway -- will rework, thanks for noting that.